prototypes/alleleFrequencies/renderAlleles.py

```diff
@@ -0,0 +1,113 @@
+import vcfAlleleLibrary
+import matplotlib.pyplot as plt
+import numpy as np
+
+"""Given a list of ExAC alleles, a bar chart of allele frequencies is plotted, with 
+subpopulations on the x axis and frequencies on the y axis"""
+
+
+def plotAlleles(alleleList):
+    """Given an alleleList, will produce 2 bar charts rendering allele freq
+    data"""
+    figureNum = 0 #will allow "figure 1" and "figure 2" to be made
+    for allele in alleleList: #allele is an object with attributes
+        figureNum += 1
+        if len(allele.pops) == 5:
+            color = 'cyan'
+            title = '1000 Genomes Frequencies'
+            plotAllele(allele, title, color)
+            truncPlot(allele, title, color)
+        if len(allele.pops) == 7:
+            color = 'salmon'
+            title = 'ExAC Frequencies'
+            plotAllele(allele, title, color)
+            truncPlot(allele, title, color)
+        else:
+            print 'Vcf.gz not recognized'
+
+
+def plotAllele(allele, title, color):
+    """given an allele, will create a bar chart for its allele frequencies"""
+    labels, AFs = makeFrequencyVectors(allele)
+    sigLine = np.full(len(labels), 0.05)
```

Comment:
Suggestion: rather than hard-coding this parameter to 0.05 (which I've just learned should actually be 0.01), pass it in as an optional parameter, e.g. sigThreshold=0.01).